configs/examples/rbac_cel.yaml

```diff
@@ -0,0 +1,176 @@
+# RBAC with policies based on CEL expressions
+# curl -v "localhost:7777" will proxy to localhost:9901, the admin page
```

Comment:
I was suggesting putting this in https://github.com/envoyproxy/envoy/tree/main/examples rather than `configs/examples`, as a full fledged example with Docker support.

Reply:
It seems like the pr has to be accepted first for that to happen as the examples pull the latest envoy image.  I will add to examples once this pr is accepted.